src/components/navs/AppNav/AppNavActivityBtn/ActivityRows.vue

```diff
@@ -68,4 +68,10 @@ export default defineComponent({
 .row:last-child {
   @apply mb-0;
 }
+.summary {
+  @apply overflow-hidden;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
```

Comment:
css magic by @pkattera

Reply:
What does it do?

Reply:
kinda like a multi-line support for `text-overflow: ellipsis`